lib/credentials/RoleCredentials.js

```diff
@@ -1,3 +1,4 @@
+const { ArsenalError } = require('arsenal/build/lib/errors');
```

Comment:
with next arsenal release, we should be able to import ArsenalError cleanly: c.f. [scality/Arsenal@`2b5d09a` (#2285)](https://github.com/scality/Arsenal/pull/2285/commits/2b5d09abb7f62b39d60a771039fe1a9d0be9dfb1)